book.rb

```diff
@@ -0,0 +1,10 @@
+# Student class inherits from Person class
+class Book
+  attr_accessor :title, :author, :rentals
+
```

Comment:
- [ ] Similarly, we need to explicitly define a method for adding rentals in _Book_ class for this activity. This should be similar to:
```ruby
class Book
  attr_accessor :title, :author, :rentals
  .
  .

  def initialize(title, author)
    .
    .
    @rentals = []
  end
  
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```
You can read further on the same via this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)